Anixe.QualityTools/Anixe.QualityTools.csproj

```diff
@@ -18,10 +18,10 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="BenchmarkDotNet" Version="0.11.5" />
-    <PackageReference Include="ConsoleMenu-simple" Version="2.2" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0" />
-    <PackageReference Include="xunit" Version="2.4" />
-    <PackageReference Include="FluentAssertions" Version="5.6" />
-    <PackageReference Include="FluentAssertions.Json" Version="5.0" />
+    <PackageReference Include="ConsoleMenu-simple" Version="2.4.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.*" />
+    <PackageReference Include="xunit" Version="2.4.1" />

```

Comment:
here you can also put a wildcard, in other references also

Reply:
Replaced